server/project.go

```diff
@@ -221,6 +221,13 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusNotFound, fmt.Sprintf("Project not found with ID %d", projectID))
 		}
 
+		if project.SchemaMigrationType == api.ProjectSchemaMigrationTypeSDL {
+			// FIXME(jc): We do not yet support file path template for state-based
+			// migration, thus we cheat here to pass validations and to avoid major
+			// refactoring for MVP.
+			repositoryCreate.FilePathTemplate = "{{DB_NAME}}__{{VERSION}}__{{TYPE}}.sql"
```

Comment:
As a second thought, we still need the file path template for data change migration.

Reply:
Yes, we will support it in subsequent PRs (https://linear.app/bbteam/issue/BYT-1233/sbm-support-filepathtemplate).